uint32 car_id
---
bool inside_nav_flag